runner/runner.go

```diff
@@ -152,7 +176,33 @@ func (r *Runner) RunAction(a []actions.Action, data map[string]any) string {
 }
 
 func (r *Runner) handleNotifications() {
+	b := retry.NewFibonacci(1 * time.Second)
+	b = retry.WithJitter(500*time.Millisecond, b)
+	b = retry.WithMaxRetries(10, b)
+
 	for n := range r.notifications {
-		r.log.Info("got notification", "notification", n)
+		n := n // pin in loop
```

Comment:
This might be a good learning oppertunity for me/you:
As far as I understood, this line should not be nessesary in modern go 🤔

https://go.dev/blog/loopvar-preview

> For Go 1.22, we plan to change for loops to make these variables have per-iteration scope instead of per-loop scope.

Reply:
```suggestion
```